docs/authorization.md

```diff
@@ -0,0 +1,156 @@
+# Relativity Trace Authorization
+
+This document lists various methods of authorization with Azure services including Exchange Web Services (EWS) and Azure Information Protection (AIP).
+
+### Exchange Web Services (EWS)
+
+Trace offers three methods for authentication and authorization with Exchange: Basic Authentication and 2 forms of OAuth 2.0 using application registrations in Azure.
+
+##### Basic Authentication

```

Comment:
@cpvanfossan Should we call out that this is being deprecated in October? https://developer.microsoft.com/en-us/graph/blogs/upcoming-changes-to-exchange-web-services-ews-api-for-office-365/ I don't want it to appear like this is a good option when it's going to be going away in a couple months.